pager-indicators/src/main/java/com/google/accompanist/pager/PagerIndicator.kt

```diff
@@ -77,6 +80,105 @@ fun HorizontalPagerIndicator(
     spacing: Dp = indicatorWidth,
     indicatorShape: Shape = CircleShape,
 ) {
+    val stateBridge = remember(pagerState) {
+        object : PagerStateBridge {
+            override val currentPage: Int
+                get() = pagerState.currentPage
+            override val currentPageOffset: Float
+                get() = pagerState.currentPageOffset
+        }
+    }
+
+    HorizontalPagerIndicator(
+        pagerState = stateBridge,
+        pageCount = pageCount,
+        modifier = modifier,
+        pageIndexMapping = pageIndexMapping,
+        activeColor = activeColor,
+        inactiveColor = inactiveColor,
+        indicatorHeight = indicatorHeight,
+        indicatorWidth = indicatorWidth,
+        spacing = spacing,
+        indicatorShape = indicatorShape
+    )
+}
+
+/**
+ * An horizontally laid out indicator for a [HorizontalPager] or [VerticalPager], representing
+ * the currently active page and total pages drawn using a [Shape].
+ *
+ * This element allows the setting of the [indicatorShape], which defines how the
+ * indicator is visually represented.
+ *
+ * @sample com.google.accompanist.sample.pager.HorizontalPagerIndicatorSample
+ *
+ * @param pagerState The Compose Foundation PagerState object of your [Pager] to be used to observe
+ * the list's state.
+ * @param modifier the modifier to apply to this layout.
+ * @param pageCount the size of indicators should be displayed.
+ * If you are implementing a looping pager with a much larger [pageCount]
+ * than indicators should displayed, e.g. [Int.MAX_VALUE], specify you real size in this param.
+ * @param pageIndexMapping describe how to get the position of active indicator by the giving page
+ * from [PagerState.currentPage].
+ * @param activeColor the color of the active Page indicator
+ * @param inactiveColor the color of page indicators that are inactive. This defaults to
+ * [activeColor] with the alpha component set to the [ContentAlpha.disabled].
+ * @param indicatorWidth the width of each indicator in [Dp].
+ * @param indicatorHeight the height of each indicator in [Dp]. Defaults to [indicatorWidth].
+ * @param spacing the spacing between each indicator in [Dp].
+ * @param indicatorShape the shape representing each indicator. This defaults to [CircleShape].
+ */
+@OptIn(ExperimentalFoundationApi::class)
+@ExperimentalPagerApi
+@Composable
+fun HorizontalPagerIndicator(
+    pagerState: FoundationPagerState,
```

Comment:
it is ok to have such aliases in test files, but in the api and public function it is better to fully spell it out. our user can see it and think that there is a class with such name and be confused